### PR TITLE
Queue task details | Link to Hearing Worksheet

### PR DESCRIPTION
**Pull request by dannysellers**
Connects #4363 

## Acceptance Criteria
- [ ] If the appeal's hearing was held and it has a Hearing Worksheet established by Hearing Prep, a link to Hearing Worksheet exists in the Appeal Summary section (see mocks below for placement)
- [ ] Hearing Worksheet opens in a new tab (this is the same convention for Reader Welcome Gate and Claims folder details today)

### CODE REVIEW

File: spec/feature/queue_spec.rb [hunk 1]
```diff
@@ -161,6 +161,8 @@
         expect(page).to have_content("Hearing Preference: #{hearing.type.capitalize}")
         expect(page).to have_content("Hearing held: #{hearing.date.strftime('%-m/%e/%y')}")
         expect(page).to have_content("Judge at hearing: #{hearing.user.full_name}")
+
+        expect(page).to have_content("View Hearing Worksheet")
```
Comment (nickheiner-usds) [inline spec/feature/queue_spec.rb]: Let's add an assertion about the `href` content of the link so we can verify that it's going to the right place.

**Comment by nickheiner-usds** [comment]
Can you add test instructions for how I can see an appeal with a hearings worksheet?

**Comment by dannysellers** [comment]
On my local tasks `111112` and `222221` have hearings (i.e. [http://localhost:3000/queue/tasks/222221](http://localhost:3000/queue/tasks/222221)). `Fakes::QueueRepository.tasks_for_user("System Admin").reject { |a| a.hearings.empty? }`

There's currently no indication of which appeals have hearings in the table view.

File: client/app/queue/AppealDetail.jsx [hunk 2]
```diff
@@ -54,7 +57,10 @@ export default class AppealDetail extends React.PureComponent {
         value: StringUtil.snakeCaseToCapitalized(lastHearing.type)
       }, {
         label: 'Hearing held',
-        value: <DateString date={lastHearing.held_on} dateFormat="M/D/YY" />
+        value: <React.Fragment>
+          <DateString date={lastHearing.held_on} dateFormat="M/D/YY" style={marginRight} />
+          <Link target="_blank" href={`/hearings/${lastHearing.id}/worksheet/print`}>View Hearing Worksheet</Link>
```
Comment (nickheiner-usds) [inline client/app/queue/AppealDetail.jsx]: Do we really want the print view?

When I visit this link locally, the fake data does not match up. The worksheet is for a different appellant. I'm going to assume that's a fake data issue.

Comment (dannysellers) [reply client/app/queue/AppealDetail.jsx]: Good point.

I think fake data is generated differently in different apps—I also get randomized vet data every time I reload the Intake page. The VACOLS ID does seem to persist, though: If you click the Review Claims Folder button at the bottom of the page, it takes you to the same URL as going from the task to Reader.

**Pull request merged**